src/main/kotlin/com/terraformation/backend/documentproducer/VariableService.kt

```diff
@@ -75,6 +75,7 @@ class VariableService(
               }
 
               variableValueStore.upgradeSectionDefaultValues(replacements)
+              variableValueStore.upgradeSectionValueVariables(replacements)
```

Comment:
We should call this in `importAllVariables` too, I'm pretty sure. `upgradeAllVariables` is called when you hit the "Upgrade Old Variable Values" button in the admin UI, not when you upload a new CSV.